5.machine_learning/scripts/3.visualize_model_coef.r

```diff
@@ -120,6 +129,44 @@ print(dim(failing_coef_df))
 head(failing_coef_df, 2)
 
 
+# Load and clean failing class data frame
+ranked_coef_df <- readr::read_csv(
+    ranked_coef_file, col_types = readr::cols(.default = "d", Feature = "c")
+)
+
+# Split out components of feature name for visualization
+ranked_coef_df <- ranked_coef_df %>%
+    dplyr::arrange(desc(abs(Coefficient))) %>%
+    tidyr::separate(
+        Feature,
+        into = c(
+            "compartment",
+            "feature_group",
+            "measurement",
+            "channel", 
+            "parameter1", 
+            "parameter2",
+            "parameter3"
+        ),
+        sep = "_",
+        remove = FALSE
+    ) %>%
+    dplyr::mutate(channel_cleaned = channel)
+
+ranked_coef_df$channel_cleaned <- dplyr::recode(ranked_coef_df$channel_cleaned,
+    "Hoechst" = "Nucleus",
```

Comment:
Would there be any information loss with the change from `Hoechst` to `Nucleus` here? For example, does `Hoechst` signal always mean `Nucleus` or is there a nuance here that could be important to the audience?